--- Fingerprint.swift
+++ Fingerprint.swift
@@ -242,6 +242,7 @@
 class Secret {
 
     private static let keyName: String = "__aio_key"
+    private static let accessGroup = "group.local.app.padloc"
 
     private func getBioSecAccessControl(invalidateOnEnrollment: Bool) -> SecAccessControl {
         var access: SecAccessControl?
@@ -273,6 +274,7 @@
         let query: [String: Any] = [kSecClass as String: kSecClassGenericPassword,
                                     kSecAttrAccount as String: Secret.keyName,
                                     kSecAttrAccessControl as String: getBioSecAccessControl(invalidateOnEnrollment: invalidateOnEnrollment),
+                                    kSecAttrAccessGroup as String: Secret.accessGroup,
                                     kSecValueData as String: password]
 
         let status = SecItemAdd(query as CFDictionary, nil)
@@ -285,6 +287,7 @@
                                     kSecMatchLimit as String: kSecMatchLimitOne,
                                     kSecReturnData as String : kCFBooleanTrue,
                                     kSecAttrAccessControl as String: getBioSecAccessControl(invalidateOnEnrollment: true),
+                                    kSecAttrAccessGroup as String: Secret.accessGroup,
                                     kSecUseOperationPrompt as String: prompt]
 
         var item: CFTypeRef?
@@ -303,6 +306,7 @@
 
     func delete() throws {
         let query: [String: Any] = [kSecClass as String: kSecClassGenericPassword,
+                                    kSecAttrAccessGroup as String: Secret.accessGroup,
                                     kSecAttrAccount as String: Secret.keyName]
 
         let status = SecItemDelete(query as CFDictionary)
